services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/HankeControllerTest.kt

```diff
@@ -94,29 +94,38 @@ class HankeControllerTest {
 
     }
 
-    // sending of sub types
-/* in the construction
-
+    //sending of sub types
     @Test
-    fun `test that update with listOfOmistaja can be sent to controller and is responded with 200`() {
-        var partialHanke = Hanke(id = 123, hankeTunnus = "id123",
+    fun `test that create with listOfOmistaja can be sent to controller and is responded with 200`() {
+        var hanke = Hanke(id = null, hankeTunnus = null,
                 nimi = "hankkeen nimi", kuvaus = "lorem ipsum dolor sit amet...", onYKTHanke = false,
                 alkuPvm = getCurrentTimeUTC(), loppuPvm = getCurrentTimeUTC(), vaihe = Vaihe.OHJELMOINTI, suunnitteluVaihe = null,
                 version = 1, createdBy = "Tiina", createdAt = getCurrentTimeUTC(), modifiedBy = null, modifiedAt = null, saveType = SaveType.DRAFT)
 
+
+        hanke.omistajat = arrayListOf(
+                HankeYhteystieto(null, ContactType.TOTEUTTAJA, "Pekkanen", "Pekka",
+                        "pekka@pekka.fi", "3212312", null,
+                        "Kaivuri ja mies", null, null, null,
+                        null, null))
+
+        var mockedHanke = hanke.copy()
```

Comment:
What is the meaning of this `mockedHanke`?

Reply:
I want to add the ids for the returned object but not have them in the content that is sent for actual logic.